### PR TITLE
Developer job description

### PR DESCRIPTION
**Pull request by gsong**
@dequency/dequencers I'd appreciate a review by Wednesday, 2022-10-05.

### CODE REVIEW

**Comment by frago12** [comment]
This is a job description I'll be interested in applying to 👌. I'll do a second pass next week.

File: hr/web developer job description.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Build web apps (mostly) and smart contracts (rarely) as we experiment in the web3 space
+- Bring new development tool and process ideas
+- Help define the products by working with _everyone_ in the company
+
+We are looking for expertise from browser UI to devops, so if your skills fall somewhere in that spectrum, let's talk. We do prefer candidates who are more toward the front end of the stack.
+
+### Tools We Use
+
+- Semantic HTML
+- CSS (we currently use Tailwind, but that may not be the case going forward)
```
Comment (frago12) [inline hr/web developer job description.md]: We are not using Tailwind 😅 We are using `@mui/material`

**Comment by keatlyKHZ** [comment]
This looks great to me! I'm good with it

File: hr/web developer job description.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### How Engineering Works at Dequency
+
+- You won't be handed a solution to implement, you'll be asked to solve a problem
+- We work on weekly iterations
+- We get together once a day to show off what we've done and help each other out
+- We teach each other about what we've learned
+- We always look for better ways
+
+## What You'll Love About Dequency
```
Comment (margaretlink) [inline hr/web developer job description.md]: should we include something like "we're curious/we're passionate — we love figuring out novel ways to approach hard problems, and we enjoy being fully immersed in the web3 x music space." or something like this? I feel like there's a lot of "rest" language and I want some "work" language 😅😅😅

**Comment by margaretlink** [comment]
I love this JD! It's to-the-point and powerful. I'd be happy to share on Linkedin/boys club board etc.

File: hr/web developer job description.md (hunk 1 above)
Comment (gsong) [reply hr/web developer job description.md]: Corrected.

File: hr/web developer job description.md (hunk 2 above)
Comment (gsong) [reply hr/web developer job description.md]: @margaretlink Something like this? https://github.com/dequency/misc/pull/1/commits/3742b1198254015c6d4c3fee570eb20fdf9e2508

**Comment by gsong** [comment]
> I love this JD! It's to-the-point and powerful. I'd be happy to share on Linkedin/boys club board etc.

Feel free to post whenever you like.

**Comment by gsong** [comment]
@Carlathepoet2 See https://github.com/dequency/misc/pull/1/commits/b2f2c26c2480838c8e8cd400d9b4b8a1b074d901 for how to apply.

**Pull request merged**